src/components/BorrowerProfile/LendCta.vue

```diff
@@ -6,26 +6,29 @@
 	>
 		<div
 			:class="[
-				'tw-w-full tw-z-sticky',
+				'tw-w-full',
 				'tw-flex tw-flex-col',
-				'tw-fixed tw-left-0 tw-bottom-0',
+				'tw-left-0 tw-bottom-0',
 				{
 					'md:tw-static': !isSticky,
 				},
 				'lg:tw-static',
+				{
+					'tw-fixed tw-z-sticky': isSticky
+				}
 			]"
 		>
 			<kv-grid
 				:class="[
 					'tw-grid-cols-12',
-					'tw-px-2.5',
-					'tw-bg-primary',
-					'tw-border-t tw-border-tertiary',
 					{
 						'md:tw-rounded-b': !isSticky,
 						'md:tw-border-none': !isSticky,
 						'md:tw-px-3': !isSticky,
 						'md:tw-px-4': isSticky,
+						'tw-bg-transparent': !isSticky && isMobile,
+						'tw-bg-primary': !isMobile || isSticky,
```

Comment:
Can we apply these styles entirely with classes instead of relying on `isMobile`? I think this will work:
```
'tw-bg-transparent md:tw-bg-primary': !isSticky,
'tw-bg-primary': isSticky,
```